doc/sphinx/addendum/type-classes.rst

```diff
@@ -258,6 +258,11 @@ Substructures are components of a class which are instances of a class
 themselves. They often arise when using classes for logical
 properties, e.g.:
 
+.. deprecated:: 8.16
+
+   In future versions, in addition to the instance, this should also
+   declare a :ref:`coercion<coercions>`, like for any other record.
```

Comment:
I don't see how this relates to the material above or below it.  Can you explain or find a better place.  What does "this" refer to?

Reply:
You are right, this deserves a pointer, I added a :ref: to the appropriate paragraphs in the refman.